labgrid/driver/sshdriver.py

```diff
@@ -356,24 +356,31 @@ def forward_unix_socket(self, unixsocket, localport=None):
             yield localport
 
     @Driver.check_active
-    @step(args=['src', 'dst'])
-    def scp(self, *, src, dst):
+    @step(args=['src', 'dst', 'recursive'])
+    def scp(self, *, src, dst, recursive=False):
```

Comment:
We need to consider backwards compatibility here. So, if `src` is a string, wrap it in a list.